### DISCUSSION TITLE
[Script request] UrBackup

### DISCUSSION THREAD

**Discussion by damnsanga** [community-scripts/ProxmoxVE#562] [Request script] [answered]
### Application Name

UrBackup

### Website

https://www.urbackup.org

### Description

UrBackup is an easy to setup Open Source client/server backup system, that through a combination of image and file backups accomplishes both data safety and a fast restoration time.

File and image backups are made while the system is running without interrupting current processes.

UrBackup also continuously watches folders you want backed up in order to quickly find differences to previous backups. Because of that, incremental file backups are really fast.

Your files can be restored through the web interface, via the client or the Windows Explorer while the backups of drive volumes can be restored with a bootable USB-Stick (bare metal restore).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by tremor021** [comment, marked as answer]
#1569 Its done here.
